elastic/kibana
# [Index Management] Support Hidden Indices

## Summary

Fix https://github.com/elastic/kibana/issues/56532

Adds support for showing hidden indices in Index Management which replaces the notion of system indices in the UI.

## How to test

See the linked issue for more details, but essentially:

1. Go to Index Management
2. Toggle the show hidden indices switch
3. See some hidden indices appearing and disappearing

It would be good to test any other logic that perhaps uses the fetch_indices logic, but because our external contract has not changed this should not affect any enrich consumers.

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
